components/Feedback/Feedback.tsx

```diff
@@ -44,13 +30,37 @@ export default function PageWithJSbasedForm(props) {
 
     //Send feedback to posthog
     void sendDocsFeedback(feedback, comment);
+  };
+
+  const handleSubmit = (event: FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
 
+    if (!feedback) {
+      alert("Please select a feedback!");
```

Comment:
is this just from testing?

Reply:
Also if someone bypasses the thumbs to get to the submit button early. An alert will pop up telling them they can't do that

Reply:
like using the chrome dev console

Reply:
Can we make it console log in that case, I'm not a fan of shipping alert to production sites

Reply:
on second look I think it's ok to remove, I wasn't able to replicate the behavior I described above 👍

Reply:
Removed